src/core/include/openvino/op/squeeze.hpp

```diff
@@ -11,15 +11,19 @@ namespace op {
 namespace v0 {
 /// \brief Squeeze operation.
 ///
+/// \param torch_mode Shape inference result with dynamic rank if selected axis has 1 is in range of its dynamic
```

Comment:
This should part of constructors [see](https://github.com/openvinotoolkit/openvino/blob/7d6ffd333b2b50c1f40622660a64d5696aa460c0/src/core/include/openvino/op/concat.hpp#L26)

Reply:
Done